shell/server/proxy.js

```diff
@@ -230,6 +231,12 @@ Meteor.setInterval(() => {
 
 const proxiesByHostId = {};
 
+function parsePowerboxDescirptorList(list) {
```

Comment:
typo: `parsePowerboxDescriptorList`

ditto below where you have autocompleted the same spelling